test/primitives/array-element-test.js

```diff
@@ -195,6 +236,15 @@ describe('ArrayElement', function() {
     //     expect(items).to.have.length(4);
     //   });
     // });
+
+    describe('#clone', function() {
+      it('creates a deep clone of the element', function() {
+        var clone = arrayElement.clone();
+        expect(clone).to.be.instanceOf(minim.ArrayElement);
+        expect(clone).to.not.equal(arrayElement);
```

Comment:
> be the same instance

Since these are objects, testing equality has to be handled at the value level and not based on the instance. This is because `{ foo: 'bar'} === { foo: 'bar'}` is `false`.